test/extensions/filters/http/transformation/transformation_filter_test.cc

```diff
@@ -230,7 +230,7 @@ class TransformationFilterTest : public testing::Test {
     int decoded_data_calls = val == 0 ? 0 : 1;
     EXPECT_CALL(filter_callbacks_, addDecodedData(_, false))
         .Times(decoded_data_calls);
-    EXPECT_CALL(filter_callbacks_, clearRouteCache()).Times(0);
+    EXPECT_CALL(filter_callbacks_, downstreamCallbacks()->clearRouteCache()).Times(0);
```

Comment:
@EItanya would you happen to know how to generate these for mocks?

Reply:
I'm not sure they're generated for these cpp mocks, mostly we use the upstream ones, there may be one available for this